<launch>
  <node pkg="path_sim" type="obstacle_avoid_planning" name="obstacle_avoid_planning" output="screen" respawn="false" >
  <param name="max_velocity" value="0.6" />
  <param name="max_angular" value="0.6" />
  <param name="cmd_topic" value="/mobile_base/commands/velocity/t" />
  <param name="frequency" value="10" />
  <param name="xy_goal_tolerance" value="0.3"/>
  <param name="yaw_goal_tolerance" value="5"/>
  <param name="follow_num" value="20"/>
  </node>
  <!-- <include file="$(find path_sim)/launch/velocity_smoother.launch" /> -->
</launch>
